### DISCUSSION TITLE
How to myserver.com

### DISCUSSION THREAD

**Discussion by Darkhost1999** [jellyfin/jellyfin#10935] [Q&A]
I want to make Jellyfin accessible from a custom URL name rather than a bunch of numbers that I can but would prefer not to memorize.
I have searched for guides even in the documentation and have found none of them to explain adequately what is my desire.
If I understand correctly I need to configure a reverse proxy server and this functionality requires some major networking knowledge to perform.
Yet the Jellyfin media server client for Windows has a hint that suggests that this is much easier than I thought and I would appreciate any links, quick tips, or suggestions.
I am also game for any quick facts, tips, and information.
I use tdarr but not sure if I should use sonarr or other applications and repositories.
What plug-ins are useful? Does someone know of a plug-in to add games to Jellyfin to restore ancient Jellyfin support for streaming games?
What software do you use if not makemkv that may provide a better dump of movies?
Why can't I use 1:1 backups as ISO?

**Comment by sol8712** [comment]
This is very easy if you are accessing it locally only.
On your router you should be able to set those bunch of numbers(ip address) to a name like whatever.local:8096(or 8920?).
Log into your router and look for DHCP settings, there you should have some sort of setting for "static leases" add your servers MAC and IP address(also shown on the  router) for server that jellyfin is on.

Really the only reason you may want a reverse proxy on the server for local only is to hide the port numbers on the end of the .local site.

IF you want to make it accessible from the internet at a .com or .net site, it is a bit more complicated as there are security concerns and you should be highly confident in your setup before allowing connections from outside your local network.

**Reply by Darkhost1999** [reply]
My gateway doesn't support UPNP but my router does. (Access outside of my local network isn't interesting at this time)
My gateway has no configurations at all.
I'll keep an eye out for these settings. I have a Linksys Router if that info helps.

**Reply by btopherjohnson** [reply]
For what it's worth, most jellyfin clients will automatically detect your server if it's running locally. I wouldn't worry about memorizing your IP address or trying to mess with router settings if that's the case for you. 

If you do end up needing to memorize your IP address, you can set it to something more memorable in the router settings as people mentioned.

**Comment by JPVenson** [comment]
Hey there,
This is more a question for our forum.jellyfin.org not for a discussion on GH as its not directly related to jellyfins code.
